packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -290,15 +337,15 @@ export class ProfilesCache {
     /**
      * @deprecated Please use ProfilesCache.getProfileLoaded(...)
      */
-    public getProfileLoaded(
-        profileName: string,
-        profileType: string,
-        profile: imperative.IProfile
-    ): imperative.IProfileLoaded {
-        return ProfilesCache.getProfileLoaded(profileName, profileType, profile);
-    }
+    // public getProfileLoaded(
+    //     profileName: string,
+    //     profileType: string,
+    //     profile: imperative.IProfile
+    // ): imperative.IProfileLoaded {
+    //     return ProfilesCache.getProfileLoaded(profileName, profileType, profile);
+    // }
 
-    public static getProfileLoaded(
+    public getProfileLoaded(
```

Comment:
I see we went with the instance approach instead of a static version of this (and other methods)
That being the case, we should probably remove some of the deprecation messages.

I don't see this as an issue, unless there are outside static references to these functions.
The only place was in the `api/vscode/ZoweVsCodeExtension` which has been updated here too: 
😍